packages/react-components/theme-designer/src/useThemeDesignerReducer.ts

```diff
@@ -95,8 +105,20 @@ export const useThemeDesignerReducer = () => {
         return {
           themeLabel: 'Custom',
           brand: custom.brand,
-          theme: custom.theme,
+          theme: overrideTheme(custom.theme, custom.brand, state.overrides),
```

Comment:
i feel it'd be cleaner to not have state based on other state. just store theme and overrides in state, and combine them later. could say 'baseTheme' and 'overrides'